xbmc/music/MusicDatabase.cpp

```diff
@@ -1043,7 +1043,7 @@ bool CMusicDatabase::GetAlbum(int idAlbum, CAlbum& album, bool getSongs /* = tru
           songs.insert(idSong);
         }
 
-        int idSongArtist = record->at(songArtistOffset + artistCredit_idArtist).get_asInt();
+        record->at(songArtistOffset + artistCredit_idArtist).get_asInt();
```

Comment:
Line to be deleted. Why was it there, well probably poor editing on my part, or using it for debug and forgot to remove.